frontend/test/ui/pages/desktop/base.py

```diff
@@ -0,0 +1,17 @@
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+
+
+class Base(Page):
+
+    _copter_locator = (By.CLASS_NAME, 'copter')
```

Comment:
What's 'copter'? Class names can be obscure, but ideally page objects will be less so.